### PR TITLE
Updated phpDoc and removed duplicates

### PR DESCRIPTION
**Pull request by ASvi**
Updated phpDoc,
ValidatorPluginManager - deleted duplicate for line 103.

### CODE REVIEW

File: library/Zend/XmlRpc/Server.php [hunk 1]
```diff
@@ -492,6 +492,9 @@ public function getSystem()
      * how to handle arguments. If set to true, all methods including constructor
      * will receive the arguments. If set to false, only constructor will receive the
      * arguments
+     *
+     * @param  bool|null $flag
+     * @return Server
```
Comment (Martin-P) [inline library/Zend/XmlRpc/Server.php]: You can use `@return self` here, just like you did on the other places where `return $this;` is used.

File: library/Zend/Soap/Server.php [hunk 2]
```diff
@@ -960,7 +960,9 @@ protected function _initializeSoapErrorContext()
     /**
      * Set the debug mode.
      * In debug mode, all exceptions are send to the client.
-     * @param bool $debug
+     *
+     * @param  bool $debug
+     * @return self
```
Comment (Maks3w) [inline library/Zend/Soap/Server.php]: `@return $this`

File: library/Zend/Soap/Wsdl.php [hunk 3]
```diff
@@ -165,6 +165,7 @@ public function getClassMap()
     /**
      * Set the class map of php to wsdl mappings..
      *
+     * @param  array $classMap
      * @return self
      */
     public function setClassMap($classMap)
```
Comment (Maks3w) [inline library/Zend/Soap/Wsdl.php]: can this argument to have `array` typehint?

File: library/Zend/XmlRpc/Fault.php [hunk 4]
```diff
@@ -82,7 +82,9 @@ class Fault
 
     /**
      * Constructor
-     *
+     * 
+     * @param int|string $code
```
Comment (Maks3w) [inline library/Zend/XmlRpc/Fault.php]: string is not allowed.

**Comment by ASvi** [comment]
Updated following the latest comments

**Comment by Martin-P** [comment]
@Maks3w
Regarding `@return $this`:
In ZF2 `@return self` is used. I mentioned it earlier in [another PR](https://github.com/zendframework/zf2/pull/7444#discussion_r28588765) and referenced the comment from @weierophinney there: https://github.com/zendframework/zf2/pull/4436#issuecomment-17538536.

**Comment by ASvi** [comment]
Updated to @return self

File: library/Zend/Soap/Server.php (hunk 2 above)
Comment (weierophinney) [inline library/Zend/Soap/Server.php]: @Maks3w `@return self` is the current recommendation from PHPDocumentor.

File: library/Zend/Validator/ValidatorPluginManager.php [hunk 5]
```diff
@@ -105,7 +105,6 @@ class ValidatorPluginManager extends AbstractPluginManager
         'isfloat'                  => 'Zend\I18n\Validator\IsFloat',
         'isinstanceof'             => 'Zend\Validator\IsInstanceOf',
         'isint'                    => 'Zend\I18n\Validator\IsInt',
-        'ip'                       => 'Zend\Validator\Ip',
```
Comment (weierophinney) [inline library/Zend/Validator/ValidatorPluginManager.php]: Why is this removed?

Comment (weierophinney) [inline library/Zend/Validator/ValidatorPluginManager.php]: nm; it's because it's a duplicate.

File: library/Zend/Soap/Server.php (hunk 2 above)
Comment (Maks3w) [inline library/Zend/Soap/Server.php]: ok `$this` is part of the draft yet.
